mod/servers.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+use Friendica\Directory\Rendering\View;
+
+require_once 'include/site-health.php';
+
+function servers_content(&$a) {
+
+	$sites = array();
+
+	//Find the user count per site.
+	$r = q("SELECT `homepage` FROM `profile`");
+	if(count($r)) {
+		foreach($r as $rr) {
+			$site = parse_site_from_url($rr['homepage']);
+			if($site) {
+				if(!isset($sites[$site]))
+					$sites[$site] = 0;
+				$sites[$site] ++;
+			}
+		}
+	}
+
+	//See if we have a health for them AND they provide SSL.
+	$sites_with_health = array();
+	$site_healths = array();
+
+	$r = q("SELECT * FROM `site-health` WHERE `reg_policy`='REGISTER_OPEN' AND `ssl_state` = 1");
+	if(count($r)) {
+		foreach($r as $rr) {
+			if (isset($sites[$rr['base_url']])) {
+				$sites_with_health[$rr['base_url']] = (($sites[$rr['base_url']] / 100) + 10) * intval($rr['health_score']);
+				$site_healths[$rr['base_url']] = $rr;
+			}
+		}
+	}
+
+	arsort($sites_with_health);
+	$total = 0;
+	$public_sites = array();
+	foreach($sites_with_health as $k => $v)
+	{
+
+		//Stop at unhealthy sites.
+		$site = $site_healths[$k];
+
+		if($site['health_score'] <= 20) break;
+
+		//Skip small sites.
+		$users = $sites[$k];
+		if($users < 5) continue;
+
+		//Add health score name and user count.
+		$site['health_score_name'] = health_score_to_name($site['health_score']);
+		$site['users'] = $users;
+
+		//Figure out what this server supports.
+		$plugins = explode("\r\n", $site['plugins']);
+		$site['plugins'] = $plugins;
+		$hasPlugin = function(array $input)use($plugins){
+			return !!count(array_intersect($input, $plugins));
+		};
+
+		$site['supports'] = array(
+			'HTTPS' => $site['ssl_state'] == 1,
+			'Twitter' => $hasPlugin(array('buffer', 'twitter')),
+			'Google+' => $hasPlugin(array('buffer', 'gpluspost')),
+			'RSS/Atom' => true, //Built-in.
+			'App.net' => $hasPlugin(array('appnet', 'appnetpost')),
```

Comment:
You can remove App.net since it closed its doors. But you could add Facebook via the "buffer" addon.